docs/troubleshooting_guidence.md

```diff
@@ -390,9 +390,9 @@ summary:
   clusters: 10
 ```
 
-## Hub Backend CLI 
+## Hub Backend CLI to get the AppSub Status
```

Comment:
Since the title has changed, also need to update the link table on the top of the doc
```
- [Hub Backend CLI](#hub-backend-cli)
```